ssl/ssl_local.h

```diff
@@ -264,10 +264,13 @@ int ossl_ssl_get_error(const SSL *s, int i, int check_err);
 # endif
 
 /* Check if an SSL structure is using DTLS */
+# define SSL_IS_DTLS(s) SSL_CONNECTION_IS_DTLS(s)
 # define SSL_CONNECTION_IS_DTLS(s) \
     (SSL_CONNECTION_GET_SSL(s)->method->ssl3_enc->enc_flags & SSL_ENC_FLAG_DTLS)
```

Comment:
I assume you have a reason to add an extra level of indirection rather than untangling this directly:
```C
#define SSL_IS_DTLS(s) ((s)->method->ssl3_enc->enc_flags & SSL_ENC_FLAG_DTLS)
```

Reply:
I doubt it will matter either way, they are right next to each other. Having this enables search and replace to do a decent job.